backend-node/helpers/jwt.js

```diff
@@ -0,0 +1,34 @@
+import { tokenGenerate } from '@vonage/jwt';
+
+const appId = process.env.VCR_API_APPLICATION_ID || process.env.appId;
+const privateKey = process.env.VCR_PRIVATE_KEY || process.env.privateKey;
+
+const aclPaths = {
+  "paths": {
+    "/*/users/**": {},
```

Comment:
```suggestion
    "/*/rtc/**":{},
    "/*/users/**": {},
```
Think we need RTC?

Reply:
That I am not 100% sure of. I feel like the ACLS keep changing every week!

Reply:
Same. haha